brave/brave-core
# Use compile-time variables for uphold settings

Resolves https://github.com/brave/brave-browser/issues/14900

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- Start browser with `--rewards=staging=true`
- Verify wallet and ensure that there are no errors
- Ensure that we connected to Uphold's staging server
- Start browser with `--rewards=staging=fale`
- Verify wallet and ensure that there are no errors
- Ensure that we connected to Uphold's production server
